### DISCUSSION TITLE
New documentation

### DISCUSSION THREAD

**Discussion by Etheonor** [Etheonor/supanextail#97] [Announcements]
Hello!

I just created a new documentation for SupaNexTail. The website is available here and I used Docusaurus to do it. https://supanextail-documentation.vercel.app

Tell me what you think and if it's clear enough. It's not finished yet and I want to add a page about how to launch your website with Vercel.

**Comment by YannHulot** [comment]
Great documentation. Its clear an simple and explains what needs to be done. Thanks for writing it.
